commands/operator-sdk/cmd/new.go

```diff
@@ -86,6 +86,19 @@ func newFunc(cmd *cobra.Command, args []string) {
 	}
 	pullDep()
 	generate.K8sCodegen(projectName)
+	// c := &generator.Config{}
```

Comment:
if code is not being used, we should get rid of it?

Reply:
Yes, it is currently commented out to test on travis, if it works I will remove the code